components/docs-chef-io/content/automate/datafeed.md

```diff
@@ -33,37 +29,241 @@ A Data Feed operates by doing the following:
 
 By default, only Admin users of Chef Automate may create and manage Data Feeds.
 
-## Adding a Data Feed Instance
+## Data Feed Integration
 
-A single Data Feed instance connects to one 3rd party endpoint.
-Create as many Data Feed instances as needed.
+Data Feed instance sends client run and compliance scan data to the 3rd party integrations available. To add a Data Feed instance in Chef Automate:
 
-To add a Data Feed instance in Chef Automate:
+1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
+1. Select **New Integration**
 
-![Setup Data Feed Page](/images/automate/filled_form_create_data_feed.png)
+Currently, the data feed has two types of integrations:
 
-1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
-1. Select **Create Data Feed**
-1. Enter a unique Data Feed name
-1. Enter the URL for your Data Feed endpoint, including any specific port details
-1. Enter the Username and Password that your 3rd party endpoint requires for authentication
-1. Select **Test Data Feed** to begin validating the connection details
-1. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+- Webhook Integration
+  - ServiceNow
+  - Splunk
+  - ELK
+  - Custom
+- Storage Integration
+  - Minio
+  - Amazon S3
+
+{{< figure src="/images/automate/choose-a-data-feed-integration.png" alt="Choose a Data Feed Integration">}}
+
+## Webhook Integration
+
+Create a data feed instance using any webhook integration.
+
+### ServiceNow
+
+To add a Data Feed instance in Chef Automate using ServiceNow integration:
```

Comment:
I solved this a little bit differently in my ServiceNow PR. I think both approaches have merit. See PR #5785, the shortcode file is `components/docs-chef-io/layouts/shortcodes/servicenow_incidents.md`

Reply:
```suggestion
To add a ServiceNow data feed:
```